docs/API/Network/RequestManagerFactory.md

```
@@ -0,0 +1,64 @@
+
+## Namespace
+
+`OpenFGA\Network`
+
+## Source
+
+[View source code](https://github.com/evansims/openfga-php/blob/main/src/Network/RequestManagerFactory.php)
+
+## Methods
+
+#### create
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)